tflib/imagetest/helm/main.tf

```
@@ -26,22 +33,30 @@ variable "values" {
   default     = {}
 }
 
+variable "values_file" {
+  description = "Path to values file on the local filesystem."
+  type        = string
+  default     = ""
```

```suggestion
  type        = list(string)
  default     = []
```

imo we make this a list, since multiple `--values` is valid, and I can see us leveraging this in the future

---

Agreed here, needs a couple other things to make it work though, will push a commit

---

Since we're going to allow multiple values files I'm going to make a note about the precedence. Highest precedence will be the inline values, then the last values file in the list, the lowest precedence will be the first values file specified